plugins/gatsby-source-jenkinsplugins/utils.js

```diff
@@ -7,6 +7,7 @@ const URL = require('url');
 const axiosRetry = require('axios-retry');
 const dateFNs = require('date-fns');
 const {parseStringPromise} = require('xml2js');
+const categoryList = require('./categories.json');
```

Comment:
I'm not totally sold on plugin site being the owner of any data... I'll be honest I also don't think categories are very useful right now. How do you feel about getting the static json file inside of update center?

Reply:
I could move it there of course, but I suspect it's out of scope for update center too? @daniel-beck ideas?